customer-churn/streamlit_apps/streamlit_app_kubeflow.py

```diff
@@ -0,0 +1,99 @@
+import streamlit as st
+from zenml.repository import Repository
+
+
+def main():
```

Comment:
would love to check this out in the browser :)

Reply:
I haven't deployed this yet, but I will do so

Reply:
Me too I'd like to see how this looks like when deployed since I think I'll have some feedback then as well.